README.md

```diff
@@ -14,7 +14,26 @@ from the StellarStation Console. Details for registering call credentials on a g
 found [here](https://grpc.io/docs/guides/auth.html). Note that if the key has been revoked on the
 console, it will not be usable to authenticate with the API.
 
-// TODO(rag): Provide details on how to register call credentials for StellarStation private keys.
+The `grpc-auth` and `google-auth-library-oauth2-http` libraries can be used to easily setup
+authentication of an API client.
+
+```java
```

Comment:
Java

Reply:
This is just for Github's codeformatting of the snippet, I think it's standard to use all lowercase for it. It's not displayed.

Reply:
Never mind